packages/pretty-format/src/plugins/ImmutableMap.js

```diff
@@ -10,12 +10,17 @@
 
 'use strict';
 
-import type {Colors, Indent, Options, Print} from '../types.js';
+import type {Colors, Indent, Options, Print, Plugin} from '../types.js';
 
 const printImmutable = require('./lib/printImmutable');
 
 const IS_MAP = '@@__IMMUTABLE_MAP__@@';
-const test = (maybeMap: any) => !!(maybeMap && maybeMap[IS_MAP]);
+const IS_ORDERED = '@@__IMMUTABLE_ORDERED__@@';
+const isMap = (maybeMap: any) => !!maybeMap[IS_MAP];
+const isNotOrdered = (maybeOrdered: any) => !maybeOrdered[IS_ORDERED];
+
+const test = (maybeMap: any) => 
+  !!(maybeMap && isMap(maybeMap) && isNotOrdered(maybeMap));
```

Comment:
Can we inline this? `isMap` and `isNotOrdered` functions add micro, but unnecessary, overhead to the test function
```js
const test = (maybeMap: any) => 
  !!(maybeMap && maybeMap[IS_MAP] && !maybeMap[IS_ORDERED]);
```